src/pages/MediaManip/MediaManip.module.scss

```diff
@@ -0,0 +1,28 @@
+.mediaManip {
+  width: 80%;
+  margin: 40px auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &_title {
+    font-family: "Poppins", sans-serif;
+    color: #4e89ae;
+    font-size: 3rem;
+    font-weight: bold;
+    text-shadow: 3px 4px 2px #0000002e;
+  }
+
+  &_dropDowns {
+    margin-top: 3rem;
+    width: 70%;
+    display: flex;
```

Comment:
```suggestion
```
I think this will get inherited from the parent class i.e, `mediaManip` as it is already been declared. Can you please remove it and try?